Clarify suspend volumes and capacity (#2409)
superfly/docs: reference/suspend-resume.html.md

---

## Volume behavior with suspend

Suspend automatically saves a machine's memory state to persistent storage. You don't need to attach a [volume](/docs/volumes/) for suspend to work, and the snapshot isn't stored on a volume.

> If your machine does have a volume attached, the volume and its data aren't affected by suspend and resume. It helps to keep two things separate:

Reviewer: ```suggestion
If your machine does have an attached volume, the volume and its data aren't affected by suspend and resume. It helps to think of the machine snapshot and the volume as separate things:
```

> - The **snapshot** is the saved CPU and memory state, managed by Fly. It can be discarded (for example, when you deploy new code), which forces a [cold start](#snapshot-behavior-with-suspend).

Reviewer: ```suggestion
- The **snapshot** is the saved CPU and memory state, managed by Fly. It can be discarded, for example, during a deploy, which forces a [cold start](#snapshot-behavior-with-suspend).
```

> - Your **volume** is your own persistent storage. Its data survives suspend, resume, and cold starts, just as it does across a normal stop and start.

Reviewer: ```suggestion
- The **volume** is your persistent storage. Its data survives suspend, resume, and cold starts, just as it does across a normal stop and start.
```

Suspended machines cost the same as stopped machines: storage only. There are no CPU/RAM charges.

> If a suspended machine has a volume, you keep paying for that volume the whole time it exists. [Volume storage](/docs/about/pricing/#persistent-storage-volumes) is billed whether the machine is running, stopped, or suspended.

Reviewer: ```suggestion
If a suspended machine has a volume, you continue paying for the volume for as long as it exists [Volume storage](/docs/about/pricing/#persistent-storage-volumes) is billed whether the machine is running, stopped, or suspended.
```

<div class="note icon">

> Suspending lowers your cost, but it doesn't free up room in a region. A suspended machine still takes up capacity in the region, so it isn't a way to fit more machines into a capacity-constrained one.

Reviewer: ```suggestion
Suspending a machine lowers your costs, but it does not free capacity in a region. Suspended machines still reserve their resources, so suspension is not a way to fit more machines into a capacity-constrained region
```
